### ISSUE TITLE
Revamp Front End 

### ISSUE THREAD

**Issue by mrsaicharan1** [PaytmBuildForIndia/ResearchFLO#1]
1. Create new front end templates for the entirety of the website.
2. Prepare a base HTML design template and extend it to all the other templates.

**Comment by jeffysam6** [comment]
Is open code still running?
and I would like to work on this issue

**Comment by jeffysam6** [comment]
so I just have to create a base.html and extend it to all other html ?

**Comment by mrsaicharan1** [comment]
Yes, but the base template needs to possess good UI. Will review your PR as soon as you submit it.
